### DISCUSSION TITLE
Can I change output filename?

### DISCUSSION THREAD

**Discussion by mochi-yu** [microsoft/typespec#3514] [Q&A] [answered]
Hi, I am using TypeSpec to create an OpenAPI3 specification.

This OpenAPI3 specification is compiled with the filename `openapi.yaml' by default, can I change this filename?

I have consulted the Emitter options list and could not find such an option.
https://typespec.io/docs/emitters/openapi3/reference/emitter

Thanks

**Comment by timotheeguerin** [comment, marked as answer]
`output-file` is the option you are looking for I believe https://typespec.io/docs/next/emitters/openapi3/reference/emitter#output-file

**Reply by mochi-yu** [reply]
Oh, sorry. I misunderstood the meaning.
Thanks for your quick reply.
